docs/cloud-native-security/cloud-nat-sec-posture-dashboard.asciidoc

```diff
@@ -1,18 +1,18 @@
 [[cloud-nat-sec-posture-dashboard]]
 // Note: This page is intentionally duplicated by docs/dashboards/cloud-posture.asciidoc. When you update this page, update that page to match. And careful with the anchor links because they should not match.
 
-== Cloud Posture dashboard
+= Cloud Posture dashboard
 
-The Cloud Posture dashboard summarizes your Kubernetes infrastructure's overall performance against CIS <<benchmark-rules,security benchmarks>>. To learn how to collect this data from your clusters, refer to <<get-started-with-kspm, Get started with KSPM>>.
+The Cloud Posture dashboard summarizes your cloud infrastructure's overall performance against <<benchmark-rules,security guidelines>> defined by the Center for Internet Security (CIS). To get started monitoring your security posture, refer to <<cspm-get-started, Get started with Cloud Security Posture Management>> or <<get-started-with-kspm, Get started with Kubernetes Security Posture Management>>.
 
 [role="screenshot"]
 image::images/cloud-sec-dashboard.png[The cloud Security dashboard]
```

Comment:
<img width="960" alt="Screenshot 2023-03-29 at 13 46 05" src="https://user-images.githubusercontent.com/53792284/228624237-4bbd6bda-a2b4-4684-a76c-b0344a37808a.png">

Let's use the screenshot above instead ^